adapters/yssp/yssp.go

```diff
@@ -0,0 +1,213 @@
+package yssp
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type YSSPAdapter struct {
+	URI string
+}
+
+func (a *YSSPAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	errors := make([]error, 0, 1)
```

Comment:
Let's be more optimistic and assume there will be no errors, instead of assuming 1 error. Please replace with:
```
var errors []error
```